scripts/tests/scripts_regression_tests.py

```diff
@@ -178,13 +178,19 @@ def setUpClass(cls):
         cls._testroot = os.path.join(TEST_ROOT, 'TestUnitTests')
         cls._testdirs = []
 
+    def _has_unit_test_support(self):
```

Comment:
Note that scripts_regression_tests will now run the Fortran unit tests on any machine for which PFUNIT_PATH is defined for the default compiler